docs/upgrade/upgrade-security.asciidoc

```diff
@@ -32,37 +44,54 @@ To re-enable your rules from the Rules page:
 
 . Go to the All rules table (*Detect -> Rules*).
 . Select the rules that you want to enable.
-. Click *Bulk actions -> Activate Selected* to re-enable the rules.
+. Click *Bulk actions -> Enable* to re-enable the rules.
 
-Alternatively, you can use the <<bulk-actions-rules-api, Bulk rule actions>> API to re-enable rules.
+[float]
+[[reenable-rules-upgrade]]
+=== Re-enable disabled rules
 
-*Upgrade {agent}s*
+IMPORTANT: The following applies to customers who are upgrading from 7.17 to 8.0.1 and newer only. If you are upgrading to 8.0., refer to the instructions above for <<track-rules-upgrade, tracking rules prior to upgrading to 8.0.>>.
 
-Upgrade your {stack} and {agent}s to 7.17 first (refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Fleet-managed Elastic Agents]). After this is complete, you can  {stack-ref}/upgrading-elastic-stack.html[upgrade the {stack}] to 8.0. Initially, {agent}s will be version 7.17; this is fine because {elastic-sec} supports the last minor release in 7.x (7.17) and any subsequent {elastic-endpoint} versions in 8.x. After the {stack} upgrade, you can decide whether to upgrade {agent}s to 8.0, which is recommended to ensure you get the latest features.
+Any rules that are active when you upgrade from 7.17 to 8.0 are automatically disabled, and a tag named `auto_disabled_8.0` is added to those rules for tracking purposes. Once the upgrade is complete, you can filter rules by the newly added tag, then use bulk actions to re-enable them:
```

Comment:
```suggestion
Any rules that are active when you upgrade from 7.17 to 8.0.1 or newer are automatically disabled, and a tag named `auto_disabled_8.0` is added to those rules for tracking purposes. Once the upgrade is complete, you can filter rules by the newly added tag, then use bulk actions to re-enable them:
```

Reply:
You already mentioned this in the admonition above, but I don't think it'd hurt to mention a second time.